public/locales/en.json

```
@@ -174,7 +174,11 @@
     "members": "Members",
     "joinNow": "Join Now",
     "joined": "Joined",
-    "withdraw": "Widthdraw"
+    "withdraw": "Widthdraw",
+    "orgJoined": "Joined organization successfully",
+    "MembershipRequests": "Membership request sent successfully",
```

```suggestion
    "MembershipRequestSent": "Membership request sent successfully",
```
I think it would be better to name this as `MembershipRequestSent` considering the other variable names.

---

Same for other `public/locales` files.

---

yeah sure